runtime/src/accounts_db.rs

```diff
@@ -2458,6 +2482,11 @@ impl AccountsDB {
             let pubkey = pubkey_account.0;
             self.accounts_index
                 .upsert(slot, pubkey, info, &mut reclaims);
+            self.accounts_index.update_secondary_indexes(
```

Comment:
maybe, we need to consider forks (unrooted ones could be abondoned) much like AccountsIndex?

Reply:
hmm, I think this is already handled?, so my above comment is wrong:  https://github.com/solana-labs/solana/pull/14212/files?diff=split&w=1#diff-db6927be895a2b4d140c54b84392a861b4535e91afc466c5e93c243c6bf2b28aR443

Reply:
hehe yeah, I had a much more complicated version that took those into account, but moved to this simpler fix for the immediate band-aid

Reply:
@ryoqun updated to track the slots